ui/package.json

```diff
@@ -22,6 +22,8 @@
     "test": "jest"
   },
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/types": "^5.1.0",
```

Comment:
Why are we pulling in `@material-ui/types`?

Reply:
^ Bump to get this addressed before we merge.

Reply:
TIL that material ui has its own types!! Will remove